### PR TITLE
style header column tables

### PR DESCRIPTION
**Pull request by ajluk**
Issue:  #1192
How it looks: 
Screenshot from 2019-10-21 13-32-19

### CODE REVIEW

File: styles/books/apbiology/book.scss [hunk 1]
```diff
@@ -1,5 +1,13 @@
 @import '../biology/book';
 
+@include add_settings(( 
+  HeaderColumnTable: (
```
Comment (helenemccarron) [inline styles/books/apbiology/book.scss]: per https://app.zenhub.com/workspaces/textbook-developers-5cdc84baee941c07853fbbf2/issues/openstax/cnx-recipes/1241 can we rename this to `ColumnHeaderTable`?

File: styles/designs/carnival/parts/_table-components.scss [hunk 2]
```diff
@@ -13,7 +13,13 @@ $Table: (
         border-collapse: collapse,
     )
 );
-
+$HeaderColumnTable: (
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_table-components.scss]: Can we rename this to `ColumnHeaderTable`?

File: styles/designs/carnival/parts/_table-shapes.scss [hunk 3]
```diff
@@ -37,6 +37,31 @@
         )),
     )
 )); 
+@include create_shape('carnival:::HeaderColumnTableShape',  (
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_table-shapes.scss]: Can we rename this to `ColumnHeaderTable`?

File: styles/designs/carnival/parts/_table-components.scss [hunk 4]
```diff
@@ -13,7 +13,13 @@ $Table: (
...
+    _name: "HeaderColumnTable",
+    _subselector: " > table.column-header:not(.unstyled)",
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_table-components.scss]: is `.column-header.unstyled` ever a possibility? This seems counter productive. @lwtchu might know if that is the case.

File: styles/designs/carnival/parts/_table-components.scss [hunk 5]
```diff
@@ -64,6 +70,14 @@ $TableRow: (
     _subselector: " > tr",
     _properties: ()
 );
+// $HeaderColumn: (
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_table-components.scss]: Can this be removed? If not, please leave a comment to explain why we kept this block.

File: styles/designs/carnival/parts/_table-shapes.scss [hunk 6]
```diff
@@ -37,6 +37,31 @@
...
+    _components: (
+        map-merge($TableContainer, (
+            _components: (
+                map-merge($HeaderColumnTable, (
+                    _components: (
+                        // $HeaderColumn,
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_table-shapes.scss]: Can this be removed?

File: styles/books/apbiology/book.scss (hunk 1 above)
Comment (ajluk) [reply styles/books/apbiology/book.scss]: I changed the name as suggested and removed unused commented code.

File: styles/designs/carnival/parts/_table-components.scss (hunk 4 above)
Comment (ajluk) [reply styles/designs/carnival/parts/_table-components.scss]: I removed the unstyled class.

**Pull request merged**